superset-frontend/src/components/Menu/MenuObject.tsx

```diff
@@ -44,17 +45,12 @@ export default function MenuObject({
   if (url) {
     return (
       <NavItem eventKey={index} href={url}>
-        <i className={`fa ${icon}`} /> &nbsp; {label}
+        {label}
       </NavItem>
     );
   }
 
-  const navTitle = (
-    <>
-      <i className={`fa ${icon}`} />
-      &nbsp; {label}
-    </>
-  );
+  const navTitle = <>{label}</>;
```

Comment:
this can simply be `const navTitle = label;`? Or we can remove `navTitle` entirely and use `label` instead